File: 10-Nov-Noon (Lambda)/lambda-publish-sns-topic.py
import boto3
from botocore.exceptions import ClientError
print('Loading function')

def lambda_handler(event, context):

    # Create an SNS client
    client = boto3.client('sns', region_name=AWS_REGION)

    try:
        # Publish a simple message to the specified SNS topic
        response = sns.publish(
            TopicArn='arn:aws:sns:region:0123456789:my-topic-arn',    
            Message='Hello World!',    
        )

    # Error handling
    except ClientError as e:
        print(e.response['Error']['Message'])
    else:
        print("Email sent! Message ID:"),
        print(response['MessageId'])

File: 10-Nov-Noon (Lambda)/lambda-publish-sns-topic.js
console.log("Loading function");

var AWS = require("aws-sdk");

exports.handler = function (event, context) {
	var eventText = JSON.stringify(event, null, 2);

	console.log("Received event:", eventText);

	var sns = new AWS.SNS();

	var params = {
		Message: "An object is added in SQS",

		Subject: "Message from SNS",

		TopicArn: "arn:aws:sns:us-east-1:1234567890:MyDemoTopic" //Give the ARN of your SNS
	};

	sns.publish(params, context.done);

	console.log("Published messages to SNS Topic");

};